spec/models/local_authority_spec.rb

```diff
@@ -144,4 +144,54 @@
         .to_not(change { local_authority.broken_link_count })
     end
   end
+
+  describe "External Content" do
+    context "with an active council" do
+      it "should update the external content when altered" do
+        WebMock.reset!
+        subject = build(:local_authority, content_id: SecureRandom.uuid)
+        stub_publishing_api_for_subject(subject)
+        subject.save!
+        WebMock.reset!
+
+        subject.homepage_url = "https://www.example.com/active-council"
+        stubs = stub_publishing_api_for_subject(subject)
+        subject.save!
+
+        expect(stubs.first).to have_been_requested.once
+        expect(stubs.last).to have_been_requested.once
```

Comment:
Are these testing the same thing? i.e. just line 158 and not that line 153 is also being executed?

Reply:
I love the tests though 🎉

Reply:
stubs (returned from the stub_publishing_api_for_subject) is an array (because the underlying test helper from gds-api-adapters stubs all three of the apis potentially required to publish an item: 0 = put the item, 1 = add links, 2 = publish. So we want to test that it both created and published the item (0 and 2, or first and last). We don't create links. I could test that that one isn't called, but that feels like overkill.